.github/workflows/iroha2-dev.yml

```diff
@@ -2,7 +2,7 @@ name: I2::Dev::Publish
 
 on:
   push:
-    branches: [main]
+    branches: [ testnet/2.0.0-rc.1, main]
```

Comment:
Note that after merge, published images will take `iroha:dev` tags @BAStos525

Reply:
Maybe we can publish new git tag to have `2.0.0-rc.1.0` tag?

Reply:
I'm cautious about overriding `iroha:2.0.0-rc.1.0` tag because this change would be specific to this testnet. I'd tag `iroha:testnet-2.0.0-rc.1`